parallel_report.py

```diff
@@ -1,35 +1,49 @@
 import re
-import subprocess
+import os
+import glob
+from typing import Dict, Tuple, List
 
 
-test_report = {
-    "passing": 0,
-    "pending": 0,
-    "failing": 0
-}
 
-skipped_files = []
+def process_test_suite_results() -> Tuple[Dict[str, int], List[str]]:
 
-stdout_files = subprocess.check_output('find openzeppelin-contracts/pout -name "stdout" | sort', shell=True).decode().splitlines()
+    test_report  = {
+        "passing": 0,
+        "pending": 0,
+        "failing": 0
+    }
 
-for stdout in stdout_files:
-    with open(stdout, 'r+') as f:
-        rep = f.read()
-        result = re.findall(r"(\d+) (passing|pending|failing)", rep)
-        if not result:
-            skipped_files.append(stdout)
-        for count in result:
-            test_report[count[1]] += int(count[0])
+    skipped_files = []
 
-print(f"Count of files: {len(stdout_files)}\n")
-print("Summarize result:\n")
-for state in test_report:
-    print(f"    {state.capitalize()} - {test_report[state]}")
-print(f"\nTotal tests - {sum(test_report.values())}\n")
+    script_path = os.path.dirname(os.path.realpath(__file__))
+    stdout_files = glob.glob(script_path + "/**/stdout", recursive=True)
+    print("`stdout` files found: {}. Processing ...\n".format(str(len(stdout_files))))
```

Comment:
`str(len(stdout_files))` ? Is it really needed?)

Reply:
Of course it isn't needed any more

Reply:
soved